CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ If the contribution doesn't meet these criteria, a maintainer will discuss it wi
 * Make your changes
 * When committing your changes, make sure to start the commit message with `#<issue-number>` (e.g. `git commit -m '#12 - New Feature added'`)
 * Push your new branch to your own fork into the same remote branch (e.g. `git push origin 12-new-feature`)
-* On Gitlab goto the [pull request page](https://guides.github.com/activities/forking/#making-a-pull-request) on your own fork and create a merge request to this reposistory
```

Comment:
Looool, accidental gitlab

Reply:
This was already fixed, but havent pushed that yet haha. Good catch tho :D